### PR TITLE
rjpcomputing.LuaForWindows version 5.1.5-52

### PR DESCRIPTION
**Pull request by A-Metal-Penguin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57570)

### CODE REVIEW

File: manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rjpcomputing.LuaForWindows
+PackageVersion: 5.1.5-52
+PackageLocale: en-US
+Publisher: rjpcomputing
+PackageName: LuaForWindows
```
Comment (OfficialEsco) [inline manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.locale.en-US.yaml:8]:
```suggestion
Publisher: The Lua for Windows Project and Lua and Tecgraf, PUC-Rio
PackageName: Lua for Windows
```

File: manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: rjpcomputing.LuaForWindows
+PackageVersion: 5.1.5-52
```
Comment (OfficialEsco) [inline manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.installer.yaml:5]:
```suggestion
PackageVersion: 5.1.5.52
```

File: manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rjpcomputing.LuaForWindows
+PackageVersion: 5.1.5-52
```
Comment (OfficialEsco) [inline manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.1.5.52
```

File: manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: rjpcomputing.LuaForWindows
+PackageVersion: 5.1.5-52
```
Comment (OfficialEsco) [inline manifests/r/rjpcomputing/LuaForWindows/5.1.5-52/rjpcomputing.LuaForWindows.yaml:5]:
```suggestion
PackageVersion: 5.1.5.52
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You'll need to change the Version folder name when fixing the PackageVersion, you can do this in VScode Online by pressing `.`

The Publisher and PackageName field needs to match the `A`dd or `R`emove `P`rograms table
```
Enter a pull request number: #57570

Manifest validation succeeded.
Found LuaForWindows [rjpcomputing.LuaForWindows] Version 5.1.5-52
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/rjpcomputing/luaforwindows/releases/download/v5.1.5-52/LuaForWindows_v5.1.5-52.exe
  ██████████████████████████████  27.7 MB / 27.7 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name        : Lua for Windows 5.1.5-52
Publisher   : The Lua for Windows Project and Lua and Tecgraf, PUC-Rio
Version     : 5.1.5.52
ProductCode : Lua_is1
Uninstall   : winget uninstall --id "Lua_is1"
```

**Comment by A-Metal-Penguin** [comment]
> You'll need to change the Version folder name when fixing the PackageVersion, you can do this in VScode Online by pressing `.`
> 
> The Publisher and PackageName field needs to match the `A`dd or `R`emove `P`rograms table

I believe I made these changes. I apologize if this isn't true, I am new to all this. Please review and correct me if I am wrong.

**Comment by A-Metal-Penguin** [comment]
Closed due to not understanding how to update pull request. Will be re-submitting with suggested changes.

**Comment by OfficialEsco** [comment]
Hehe, since this is good knowledge to have ill leave 2 guides
1. To submit multiple changes you can click on the [Files Changed](https://github.com/microsoft/winget-pkgs/pull/57570/files) tab
2. You will then see a Add to batch button next to the changes, add the ones you want to add
3. To submit the changes scroll all the way up and you'll see the Submit batch change next to  0/3 files viewed
----
1. To change a folder name you press `.` on your keyboard to open VScode online, this will open a web version of VSCode
2. On the left side you'll see a button called Explorer, this works like File Explorer, most of the time it will open the changed files folder, if not browser your way to the files
3. Right click the folder you want to rename, in this case the 5.2.5-52 folder, rename it to 5.2.5.25
4. When you are done editing the files you'll click the Source Control on the left side 2 below the Explorer button
5. Type something in the Message field (whats changed) and then click the ✓ button to commit the changes.

**Pull request closed without merging**